src/renderer/components/+cluster/cluster-no-metrics.module.scss

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+.ClusterNoMetrics {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  line-height: 1.7;
+  flex-grow: 1;
+
+  .link {
+    color: var(--blue);
```

Comment:
should this be a theme-friendly colour? I guess blue is used elsewhere too, and is already friendly to our 2 themes.

Reply:
Currently we have the same color for `--blue` for both themes defined in `lens-dark.json` and `lens-light.json`.

Anyway I would suggest to add some prefix to all themed variables to distinguish them from other css-variables and for easier lookup the roots, e.g. `--theme-blue` or `--lens-blue` (`blue` btw wrong name for themed variable, better to name for this case as `--theme-link-color` for example.